packages/api/test/e2e/rx-queries.spec.ts

```diff
@@ -51,4 +51,181 @@ describe.skip('Rx e2e queries', () => {
         done();
       });
   });
+
+  describe('with plain type', async () => {
```

Comment:
These are great, however could have stopped at promise - since underlying promise uses the rxjs interfaces, so if that works, we are (mostly) golden. There are obviously cases where the one may break and other other not, but at long as there is basic other coverage, it will be caught.

TL;DR - These are great, but probably technically not needed

Reply:
yeah, right, let me remove them